aws_lambda_powertools/utilities/data_classes/common.py

```diff
@@ -103,6 +104,10 @@ def headers(self) -> Dict[str, str]:
     def query_string_parameters(self) -> Optional[Dict[str, str]]:
         return self.get("queryStringParameters")
 
+    @property
+    def multi_value_query_string_parameters(self) -> Optional[Dict[str, List[str]]]:
+        return self.get("multiValueQueryStringParameters")
```

Comment:
```suggestion
    def multi_value_query_string_parameters(self) -> Dict[str, List[str]]:
        return self.get("multiValueQueryStringParameters") or {}
```

easier to have a sane default